datafusion/sqllogictest/test_files/timestamps.slt

```diff
@@ -2795,3 +2795,10 @@ SELECT '2000-12-01 04:04:12' AT TIME ZONE 'America/New York';
 # abbreviated timezone is not supported
 statement error
 SELECT '2023-03-12 02:00:00' AT TIME ZONE 'EDT';
+
+
+# Test current_time without parentheses
```

Comment:
While updating the code I realized I almost broke this, so I added a new test (also propose adding it to `main` here: https://github.com/apache/datafusion/pull/10509)